force-app/main/default/triggers/AnotherOpportunityTrigger.trigger

```diff
@@ -39,25 +39,40 @@ trigger AnotherOpportunityTrigger on Opportunity (before insert, after insert, b
     if (Trigger.isAfter){
         if (Trigger.isInsert){
             // Create a new Task for newly inserted Opportunities
+            List <Task> taskList  = new List <Task>();
             for (Opportunity opp : Trigger.new){
                 Task tsk = new Task();
                 tsk.Subject = 'Call Primary Contact';
                 tsk.WhatId = opp.Id;
                 tsk.WhoId = opp.Primary_Contact__c;
                 tsk.OwnerId = opp.OwnerId;
                 tsk.ActivityDate = Date.today().addDays(3);
-                insert tsk;
+                taskList.add(tsk);
+            }
+            if(!taskList.isEmpty()){
+                insert taskList;
```

Comment:
Try to remove this logic from the trigger and add it to a OpportunityTriggerHandler Class